### PR TITLE
Added Jukebox (#26736)

### PR DESCRIPTION
**Pull request by DJB1gYAPPA**
# Description

Adds super cool super nice Jukebox to the bar! Ran off .ogg files, can use any royalty free song. 

---

<details><summary><h1>Media</h1></summary>
<p>

![Example Media on] 
Screenshot 2024-08-26 061947

</details>

---

# Changelog

:cl: Kacey, Nova
- add: Added Jukebox with basic songs.

### CODE REVIEW

**Comment by VMSolidus** [comment]
image

Test fail is legitimate.

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml [hunk 1]
```diff
@@ -443,6 +443,7 @@
       - SurveillanceWirelessCameraMonitorCircuitboard
       - SurveillanceCameraWirelessRouterCircuitboard
       - ComputerTelevisionCircuitboard
+      - JukebocCurcuitBoard
```
Comment (VMSolidus) [inline Resources/Prototypes/Entities/Structures/Machines/lathe.yml:446]:
```suggestion
      - JukeboxCircuitBoard
```

**Comment by DangerRevolution** [comment]
Fill out the PR please; add a changelog and (optionally) add a video of it working.

**Comment by DJB1gYAPPA** [comment]
> Fill out the PR please; add a changelog and (optionally) add a video of it working.

I can not get a local build to work for me, however, it works for everyone else. I wont be able to add a video of it working, or a snapshot of what it looks like in game

**Review by VMSolidus — APPROVED** [review]

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
Please make a better Changlog, but overall.
Good job!

**Review by FoxxoTrystan — APPROVED** [review]

**Pull request merged**